tracing/client.go

```diff
@@ -165,7 +171,7 @@ func newClient(ctx context.Context, cfg *config.Config, token string) (*Client,
 		// Legacy HTTP client
 		log.Tracef("Using legacy HTTP client for Tracing: url=%v, auth.type=%s", u, auth.Type)
 		timeout := time.Duration(config.Get().ExternalServices.Tracing.QueryTimeout) * time.Second
-		transport, err := httputil.CreateTransport(&auth, &http.Transport{}, timeout, nil)
+		transport, err := httputil.CreateTransport(&auth, &http.Transport{}, timeout, cfgTracing.CustomHeaders)
 		if err != nil {
 			return nil, err
 		}
```

Comment:
I can't add the comment in the specific line, but I think the ctx should also need to be added in the Tempo grpc client? This would be in line 195: 
`
				clientConn, _ := grpc.Dial(grpcAddress, dialOps...)`

I think that could be replaced with something like: 

`clientConn, _ := grpc.DialContext(ctx, grpcAddress, dialOps...)`

Reply:
I made this change - see commit https://github.com/kiali/kiali/pull/7353/commits/abb5e7a8472e35784294dd9d3b00cf0eeac6ed14